rest_api/rest_api/schema.py

```diff
@@ -22,9 +22,20 @@
 PrimitiveType = Union[str, int, float, bool]
 
 
+class PipelineHyperParams(BaseModel):
```

Comment:
we might want to replace this class in future with going back to YAML config files, if they work for our use case, though probably there wont be enough time to do that before end of the course :D

Reply:
yes agreed!